### DISCUSSION TITLE
Diary-粤了（一） | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#341] [Announcements]
# Diary-粤了（一） | Zi-Zi's Journey

润了！

https://gz-metal-cell.github.io/2024/07/23/Diary-%E7%B2%A4%E4%BA%86%EF%BC%88%E4%B8%80%EF%BC%89/

**Comment by promefire** [comment]
太好看了吧

**Reply by GZ-Metal-Cell** [reply]
名胜古迹比山西差远了😭

**Comment by promefire** [comment]
为啥我F12找不到**8:05:G332** 底部的铁路图呢🧐

**Reply by GZ-Metal-Cell** [reply]
因为这个铁路图是作为一个 `<div class="G">` 的背景图存在而不是一个 `<img>` ！找到这个 DOM 后，看 `class="G"` 的 CSS 的 `background` 属性就可以看到铁路图的 url 了！
